can not import actioncable in standalone vue+vite project.

### Steps to reproduce
I'm using vue3+vite in a standalone js frontend project, and rails 6 as backend server.

when I want use actioncable in the project, some error occurred :

```
yarn add @rails/actioncable
yarn dev
```
errors output: 
```
yarn run v1.22.10
$ vite
 > node_modules/vite/dist/node/chunks/dep-972722fa.js:39166:10: error: [plugin: vite:dep-scan] Failed to resolve entry for package "@rails/actioncable". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "@rails/actioncable". The package may have incorrect main/module/exports specified in its package.json.
    39166 │     throw new Error(`Failed to resolve entry for package "${id}". ` +
          ╵           ^
    at packageEntryFailure (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:39166:11)
    at resolvePackageEntry (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:39162:9)
    at tryNodeResolve (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:38969:11)
    at Context.resolveId (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:38842:28)
    at Object.resolveId (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:51297:55)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
    at async resolve (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:51497:26)
    at async /Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:51611:34
    at async callback (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:812:28)
    at async handleRequest (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:644:30)

   node_modules/vite/dist/node/chunks/dep-972722fa.js:51601:18: note: This error came from the "onResolve" callback registered here
    51601 │             build.onResolve({
          ╵                   ~~~~~~~~~
    at setup (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:51601:19)
    at handlePlugins (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:736:23)
    at Object.buildOrServe (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:1024:7)
    at /Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:1756:17
    at new Promise (<anonymous>)
    at Object.build (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:1755:14)
    at Object.build (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:1631:51)
    at /Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:51456:54
    at Array.map (<anonymous>)
    at scanImports (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:51456:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
    at async optimizeDeps (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:73441:30)
    at async runOptimize (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:74167:48)
    at async Server.httpServer.listen (/Volumes/Workspace/shop_client_new/node_modules/vite/dist/node/chunks/dep-972722fa.js:74183:21)

   src/utils/actioncable/index.js:1:31: note: The plugin "vite:dep-scan" was triggered by this import
        1 │ import { createConsumer } from '@rails/actioncable';
          ╵                                ~~~~~~~~~~~~~~~~~~~~

error when starting dev server:
Error: Build failed with 1 error:
node_modules/vite/dist/node/chunks/dep-972722fa.js:39166:10: error: [plugin: vite:dep-scan] Failed to resolve entry for package "@rails/actioncable". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "@rails/actioncable". The package may have incorrect main/module/exports specified in its package.json.
    at failureErrorWithLog (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:1451:15)
    at /Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:1131:28
    at runOnEndCallbacks (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:921:63)
    at buildResponseToResult (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:1129:7)
    at /Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:1238:14
    at /Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:609:9
    at handleIncomingPacket (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:706:9)
    at Socket.readFromStdout (/Volumes/Workspace/shop_client_new/node_modules/esbuild/lib/main.js:576:7)
    at Socket.emit (node:events:379:20)
    at Socket.EventEmitter.emit (node:domain:470:12)
    at addChunk (node:internal/streams/readable:313:12)
    at readableAddChunk (node:internal/streams/readable:288:9)
    at Socket.Readable.push (node:internal/streams/readable:227:10)
    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```


### Expected behavior
normal run

### Actual behavior
error

### System configuration
**Rails version**:
6.1
**Ruby version**:
2.6.3

--- a/actioncable/package.json
+++ b/actioncable/package.json
@@ -2,7 +2,7 @@
   "name": "@rails/actioncable",
   "version": "7.0.0-alpha2",
   "description": "WebSocket framework for Ruby on Rails.",
-  "module": "app/javascript/action_cable/index.js",
+  "module": "app/assets/javascripts/actioncable.esm.js",
   "main": "app/assets/javascripts/actioncable.js",
   "files": [
     "app/assets/javascripts/*.js",

